tests/functional/cli/test_cli_variables.py

```diff
@@ -17,3 +17,10 @@ def test_list_project_variables(gitlab_cli, project):
     ret = gitlab_cli(cmd)
 
     assert ret.success
+
+
+def test_list_project_variables_with_path(gitlab_cli, project):
+    cmd = ["project-variable", "list", "--project-id", project.path_with_namespace]
+    ret = gitlab_cli(cmd)
+
+    assert ret.success
```

Comment:
It's probably asking a bit much but if we're making sure we don't send extra query params, we could add a test that also matches the query params like I do with headers here:

https://github.com/python-gitlab/python-gitlab/blob/main/tests/functional/cli/test_cli.py

Look for `query_param_matcher` to see how I did the other one, I think.

We can do this in a follow-up also.

Reply:
Done